restaurant-api/src/main/resources/application.properties

```diff
@@ -6,4 +6,6 @@ spring.datasource.password=password
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.show-sql=true
-jwt.secret=secret
\ No newline at end of file
+jwt.secret=secret
+csv.bucket=qqd-temp
+csv.region=us-east-1
```

Comment:
Why are we switching from us-east-2 to us-east1?

Reply:
The region switch was due to a change in our base infrastructure. Cloudfront distributions require ACM certificates from us-east-1 so the base infrastructure was switched.